### PR TITLE
docs: remove beta state of postgresql

### PR DESCRIPTION
**Pull request by hifabienne**
```[tasklist]

### Definition of Ready

- [ ] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.
```

### CODE REVIEW

File: docs/docs/support/general.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+Fully supported features are those that are guaranteed to work as intended and are fully tested by our team.
+If you encounter any issues with a fully supported feature, you can contact us, 
+and we will provide prompt resolution to any issues you may experience.
+
+### Partially Supported
+
+Partially supported features are those that we support to a certain extent, 
+but may not have been fully tested or may have some limitations.
+If you encounter issues with a partially supported feature, we will make our best effort to assist you, 
+but we may not be able to provide immediate resolution or comprehensive support.
```
Comment (mffap) [inline docs/docs/support/general.md:23]:
```suggestion
Enterprise supported features are those where we provide support only to users eligible for enterprise [support services](/docs/legal/support-services).
These features should be functional for eligible users, but may have some limitations for a broader use.

If you encounter issues with an enterprise supported feature and you are eligible for enterprise support services, we will provide a prompt response to the issues you may experience and make our best effort to assist you to find a resolution.
```

File: docs/docs/support/general.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+**Partially supported features**
+
+- Database PostgreSQL
+
+### Community Supported
+
+Community supported features are those that have been developed by our community and may not have undergone extensive testing or support from our team. 
+If you encounter issues with a community supported feature, we encourage you to seek help from our community or other online resources, where other users can provide assistance.
+Join our [Discrod Chat](https://zitadel.com/chat)
```
Comment (mffap) [inline docs/docs/support/general.md:33]:
```suggestion
- Join our [Discord Chat](https://zitadel.com/chat)
- Search [Github Issues](https://github.com/search?q=org%3Azitadel+&type=issues) and report a new issue
- Search [Github Discussions](https://github.com/search?q=org%3Azitadel+&type=discussions) and open a new discussion as question or idea
```

File: docs/docs/support/general.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+but we may not be able to provide immediate resolution or comprehensive support.
+
+**Partially supported features**
+
+- Database PostgreSQL
+
+### Community Supported
+
+Community supported features are those that have been developed by our community and may not have undergone extensive testing or support from our team. 
+If you encounter issues with a community supported feature, we encourage you to seek help from our community or other online resources, where other users can provide assistance.
```
Comment (mffap) [inline docs/docs/support/general.md:32]:
```suggestion
If you encounter issues with a community supported feature, we encourage you to seek help from our community or other online resources, where other users can provide assistance:
```

File: docs/docs/support/general.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+### Alpha
+
+The Alpha state is our initial testing phase. 
+It is available to everyone, but it is not yet complete and may contain bugs and incomplete features.
+We recommend that users exercise caution when using Alpha software and avoid using it for critical tasks, as support is limited during this phase.
+
+### Beta
+
+The Beta state comes after the Alpha phase and is a more stable version of the software. 
+It is feature-complete, but may still contain bugs that need to be fixed before release.
```
Comment (mffap) [inline docs/docs/support/general.md:49]:
```suggestion
It is feature-complete, but may still contain bugs that need to be fixed before general availability.
```

File: docs/docs/support/general.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+While it is available to everyone, we recommend that users exercise caution when using Beta software and avoid using it for critical tasks. 
+During this phase, support is limited as we focus on testing and bug fixing. 
+
```
Comment (mffap) [inline docs/docs/support/general.md:52]:
```suggestion

### General Available

Generally available features are available to everyone and have the appropriate test coverage to be used for critical tasks.
The software will be backwards-compatible with previous versions, for exceptions we will publish a [technical advisory](https://zitadel.com/docs/support/technical_advisory).
Features in General Availability are not marked explicitly.

```

File: docs/docs/support/general.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+Community supported features are those that have been developed by our community and may not have undergone extensive testing or support from our team. 
+If you encounter issues with a community supported feature, we encourage you to seek help from our community or other online resources, where other users can provide assistance.
+Join our [Discrod Chat](https://zitadel.com/chat)
+
+## Software Release Cycle
+
+It's important to note that both Alpha and Beta software can have breaking changes, meaning they are not backward-compatible with previous versions of the software.
+Therefore, it's recommended to use caution when using Alpha and Beta software, and to always back up important data before installing or testing new software versions.
+
```
Comment (mffap) [inline docs/docs/support/general.md:39]:
```suggestion

Only features in General Availability will be covered by support services.

We encourage our community to test Alpha and Beta software and provide feedback via our [Discord Chat](https://zitadel.com/chat). 

```

**Review by mffap — CHANGES_REQUESTED** [review]

File: docs/docs/support/general.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
+---
+title: General
```
Comment (mffap) [inline docs/docs/support/general.md:2]: @hifabienne regarding structure:  Can we call this page "Software release cycle"? and move the support states to https://zitadel.com/docs/legal/support-services ? 

But let's fix first the content comments.

Comment (mffap) [reply docs/docs/support/general.md:2]: Note to self: clarify enterprise supported features in self-hosted and extended sla, the default (included/excluded), and how to become eligible

**Comment by eliobischof** [comment]
We still have this critical issue https://github.com/zitadel/zitadel/issues/5389
@adlerhurst can we have a look together?

File: docs/docs/guides/manage/cloud/support.md [hunk 8]
```diff
@@ -2,6 +2,16 @@
...
+:::caution
+PostgreSQL extension is currently in [Beta](/docs/support/general#beta).
+Beta state will be removed as soon as automated tests are implemented. [Github Issue](https://github.com/zitadel/zitadel/issues/5741)
+:::
+
+:::caution
+Be aware that PostgreSQL is only [partially supported](/docs/support/general#partially-supported) by the ZITADEL Team.
+Make sure to read our [Production Guide](./self-hosting/manage/production#prefer-cockroachdb) before you decide to use it.
+:::
+
```
[the remark this answers is not in the document]
Comment (hifabienne) [reply docs/docs/guides/manage/cloud/support.md:14]: I think it should be support states not beta, right?

File: docs/docs/support/general.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+It's important to note that support may differ depending on the feature, and not all features may be fully supported. 
+We always strive to provide the best support possible for our customers and community,
+but we may not be able to provide immediate or comprehensive support for all features.
+Also the support may differ depending on your contracts. Read more about it on our [Legal Page](/docs/legal/introduction)
+
+### Fully Supported
+
+Fully supported features are those that are guaranteed to work as intended and are fully tested by our team.
+If you encounter any issues with a fully supported feature, you can contact us, 
+and we will provide prompt resolution to any issues you may experience.
```
[the remark this answers is not in the document]
Comment (hifabienne) [reply docs/docs/support/general.md:16]: Should we remove the fully here as well?

File: docs/docs/support/general.md (hunk 6 above)
Comment (hifabienne) [reply docs/docs/support/general.md:41]: Do you think it is not relevant to state that alpha and beta can have breaking changes?
And also that its recommended to use it with caution?

File: docs/docs/support/general.md (hunk 7 above)
Comment (hifabienne) [reply docs/docs/support/general.md:2]: Yes makes sense

File: docs/docs/guides/manage/cloud/support.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (mffap) [reply docs/docs/guides/manage/cloud/support.md:14]: Yes. Updated suggestion

File: docs/docs/support/general.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (mffap) [reply docs/docs/support/general.md:16]: Updated suggestion

File: docs/docs/support/general.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+
+### Enterprise Supported
+
+Enterprise supported features are those where we provide support only to users eligible for enterprise [support services](/docs/legal/support-services).
+These features should be functional for eligible users, but may have some limitations for a broader use.
+
+If you encounter issues with an enterprise supported feature and you are eligible for enterprise support services, we will provide a prompt response to the issues you may experience and make our best effort to assist you to find a resolution.
+
+**Partially supported features**
```
Comment (mffap) [inline docs/docs/support/general.md:25]:
```suggestion
**Enterprise supported features**
```

File: docs/docs/support/general.md [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+
+- Database PostgreSQL
```
Comment (mffap) [inline docs/docs/support/general.md:27]:
```suggestion
- Database PostgreSQL
- LDAP Identity Provider
- [Terraform Provider](https://github.com/zitadel/terraform-provider-zitadel)
```

File: docs/docs/support/general.md (hunk 6 above)
Comment (mffap) [reply docs/docs/support/general.md:41]: Yes. I wanted to make an addition. Does it delete something on Line 40?

**Comment by hifabienne** [comment]
@mffap Can you have a look at the sidebar js conflicts? I think you do have the better overview, and I don't want to break something.

**Comment by mffap** [comment]
> @mffap Can you have a look at the sidebar js conflicts? I think you do have the better overview, and I don't want to break something.

I've copied over a recent version of the sidebars.js

**Review by mffap — APPROVED** [review]

**Pull request merged**